app/models/subnet/ipv6.rb

```diff
@@ -23,7 +23,13 @@ def validate_ip(ip)
   end
 
   def self.supported_ipam_modes
-    [:eui64, :none]
+    [:eui64, :db, :none]
+  end
```

Comment:
Out of my curiousity (thank to my limited IPv6 knowledge), why IPv4 has `:dhcp` mode while IPv6 has `:eui64` mode, which I believe is also DHCP.